src/main/java/edu/kit/provideq/toolbox/mip/solvers/QuboMipSolver.java

```diff
@@ -0,0 +1,79 @@
+package edu.kit.provideq.toolbox.mip.solvers;
+
+import edu.kit.provideq.toolbox.Solution;
+import edu.kit.provideq.toolbox.meta.SolvingProperties;
+import edu.kit.provideq.toolbox.meta.SubRoutineResolver;
+import edu.kit.provideq.toolbox.mip.MipConfiguration;
+import edu.kit.provideq.toolbox.process.GamsProcessRunner;
+import edu.kit.provideq.toolbox.process.ProcessResult;
+import edu.kit.provideq.toolbox.process.ProcessRunner;
+import java.io.File;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Mono;
+
+/**
+ * {@link MipConfiguration#MIP} solver using QUBO Reformulation implementing gams.
+ */
+@Component
+public class QuboMipSolver extends MipSolver {
+  private final String scriptPath;
+  private final String dependencyScriptPath;
+  private final ApplicationContext context;
+  private static final int PENALTY = 100;
+
+  @Autowired
+  public QuboMipSolver(
+      @Value("${path.gams.mip-to-qubo-reformulation}") String mipToQuboReformulatorPath,
+      @Value("${path.gams.qubo-solve}") String solverScriptPath,
+      ApplicationContext context) {
+    this.scriptPath = mipToQuboReformulatorPath;
+    this.dependencyScriptPath = solverScriptPath;
+    this.context = context;
+  }
+
+  @Override
+  public String getName() {
+    return "Gams Solver for MIP";
```

Comment:
Rename according to [new standard](https://github.com/ProvideQ/toolbox-server/issues/123)

Algorithm (Technology)

For example:
`Qubo Transformation (Gams)`